### PR TITLE
Upgrade to akka-streams 1.0

### PR DESCRIPTION
**Pull request by jeantil**
Upgrades the dependency and fixes renamed types

(I need this to upgrade webdriver and use Framing to close my pending isssue with it)

cheers

### CODE REVIEW

**Comment by huntc** [comment]
Thanks for doing this. I'm on vacation this week. How soon do you need the release?

**Comment by jeantil** [comment]
It can wait for a week. I'll use a private release in the mean time :)

Thanks for your comment !

Le ven. 26 juin 2015 23:37, Christopher Hunt <notifications@github.com> a
écrit :

**Comment by jeantil** [comment]
akka-stream 1.0 has been released I updated the commit to reflect that ...

File: project/Dependencies.scala [hunk 1]
```diff
@@ -2,8 +2,8 @@ import sbt._
 
 object Version {
   val akka       = "2.3.11"
-  val akkaHttp   = "1.0-RC3"
-  val akkaStream = "1.0-RC3"
+  val akkaHttp   = "1.0-RC4"
+  val akkaStream = "1.0-RC4"
```
Comment (2m) [inline project/Dependencies.scala]: This should be 1.0.

**Comment by jeantil** [comment]
Indeed. I must have botched the PR somehow since that's what I have locally

**Comment by jeantil** [comment]
Sorry , seems like I pushed the wrong commit, the new one should be better

**Comment by ktoso** [comment]
LGTM, thanks!

**Comment by ktoso** [comment]
Realised I don't have merge rights here... weird :<

**Comment by jeantil** [comment]
akka-contrib-extra needs some extra love :)

**Comment by huntc** [comment]
Thanks for doing this. We have a dependency chain that'll be affected by this change so it'll be done in one foul swoop this week, including the merge of this.

**Comment by jeantil** [comment]
Ok, i'll monitor this and make the corresponding webdriver PR when the new version is released

**Comment by markusjura** [comment]
@jeantil Version 1.18.0 with akka streams 1.0 has been released.

**Comment by jeantil** [comment]
awesome ! thanks

**Pull request merged**